frontend/simple/views/SignUp.vue

```diff
@@ -16,23 +16,27 @@
           <div class="box centered" style="max-width:400px">
             <h2 class="subtitle">Sign Up</h2>
 
+            <a v-on:click="forwardToLogin" style="margin-left: 65%">Already a member?</a>
```

Comment:
Hmm, OK, I had this backwards. Clicking on starting a group should first take us to *login*, not *signup*, and this should ask, "Don't have an account?"